specs/latest/1.0/index.html

```diff
@@ -790,14 +797,60 @@ <h2 class="no-toc">Table of contents</h2>
                 </em>
                 </blockquote>
             </dd>
-        <dt><span class=prop-value>preferLowPowerToHighPerformance</span></dt>
+        <dt><span class=prop-value>powerPreference</span></dt>
             <dd>
-              Provides a hint to the implementation suggesting that, if possible, it
-              creates a context that optimizes for power consumption over
-              performance. For example, on hardware that has more than one
-              GPU, it may be the case that one of them is less powerful
-              but also uses less power. An implementation may choose to, and may
-              have to, ignore this hint.
+              <p>
+                Provides a hint to the user agent indicating what configuration of GPU is suitable
+                for this WebGL context. This may influence which GPU is used in a system with
+                multiple GPUs. For example, a dual-GPU system might have one GPU that consumes
+                less power at the expense of rendering performance. Note that this property is
+                only a hint and a WebGL implementation may choose to ignore it.
+              </p>
+              <p>
+                WebGL implementations use context lost and restored events to regulate power and
+                memory consumption, <em>even if</em> this attribute is not specified. <em>The
+                application must have registered
+                <code>"webglcontextlost"</code> and <code>"webglcontextrestored"</code> event
+                listeners on the canvas element before fetching a WebGLRenderingContext specifying
+                this attribute, or "default" will be used.</em>
```

Comment:
I like this rule. I thought we only talked about it informally, but I'm glad to see it in the specification.